server/utils/index.js

```diff
@@ -1,22 +1,28 @@
-export function dasherize(name) {
+function dasherize(name) {
   return ('' + name)
     .toLowerCase()
     .replace(/\s/g, '-')
     .replace(/[^a-z0-9\-\.]/gi, '')
     .replace(/\:/g, '');
 }
 
-export function nameify(str) {
+function nameify(str) {
   return ('' + str)
     .replace(/[^a-zA-Z0-9\s]/g, '')
     .replace(/\:/g, '');
 }
 
-export function unDasherize(name) {
+function unDasherize(name) {
   return ('' + name)
     // replace dash with space
     .replace(/\-/g, ' ')
     // strip nonalphanumarics chars except whitespace
     .replace(/[^a-zA-Z\d\s]/g, '')
     .trim();
 }
+
+module.exports = {
```

Comment:
@Bouncey That error usually means that the node does not know how to interpret a line of code. Doesn't sound like an issue in code but an issue with your set up. Which version of node are you running?